SPIE2016-9913-16/9913-16.tex

```diff
@@ -0,0 +1,303 @@
+We are therefore investigating a text-based serialization of WCS that could be usable as an interchange format.
+AST currently supports a number of serialization schemes including multiple dialects of FITS headers, STC-S\cite{2007ivoa.spec.1030R} for region handling\cite{2010ASPC..434..213B} and an internal text-based format.
+GWCS can serialize to FITS-WCS and also into YAML using the Advanced Scientific Data Format (ASDF)\cite{2015A&C....12..240G}.
+We have decided to investigate a common format that is based on YAML and the proposed new variant of STC, version 2.0.
+STC-2\cite{2015ivoaSTC2} is an evolution of STC that includes support for chainable mappings and seems like it could be a suitable starting point for this investigation.
+Both the Astropy and IVOA communities would benefit if such a standard could be established and it could be a significant factor in bringing together the competing distortion models currently in use in the FITS community.
+
+\subsection{Region handling}
+
+The handling of regions, such as circles and polygons, in either Cartesian pixel coordinates or spherical world coordinates or with discrete pixelized or analytic form (Table~\ref{tab:regions}), is not yet supported in Astropy core.
+The \texttt{photutils} affiliated Astropy package has some basic support for analytic regions in pixel coordinates for photometry and work has recently started on a regions package that can properly handle spherical geometry and world coordinate versions translated to pixels.
+LSST has its own spherical geometry package in C++, \texttt{lsst::sphgeom}, and we should compare implementations with the STScI spherical geometry library (mainly written in Python) that is being proposed for integration into Astropy.
+There is currently a focus within Astropy on handling region files from the DS9 application\cite{2005ASPC..347..110J} but there have also been discussions concerning adopting the IVOA STC standard\cite{2007ivoa.spec.1030R} as a region serialization format.
+STC-S is already being used to define sky regions in the VO community and is also understood by the GAIA visualization tool\cite{2009ASPC..411..575D} and Starlink software\cite{2010ASPC..434..213B,2014ASPC..485..391C}.
+If Astropy and LSST cannot share sky to pixel region handling code directly there is a possibility that STC could be adopted to allow interoperability.
+
+\begin{table}[t]
+\centering
+\caption{Four distinct types of region. This paper discusses all except discrete spherical.}
+\label{tab:regions}
+\vspace{10pt}
+\begin{tabular}{ccc}
+  Discrete Spherical & $\Longleftrightarrow$ & Discrete Cartesian \\
+  $\Updownarrow$ & &  $\Updownarrow$ \\
+  Geometric Spherical & $\Longleftrightarrow$ & Geometric Cartesian \\
+\end{tabular}
+\end{table}
+
+Astropy has no plans for a discretized efficient representation of a pixel region and this is something that LSST can contribute to the Astropy region handling package.
```

Comment:
I find "no plans" a bit grating here, as it sounds like this is something that Astropy doesn't really want. I'd rephrase this to make it sound more like contributing something they've just not got the resources to produce, rather than something they can't be bothered with.